### PR TITLE
P3 view validations

### PR DESCRIPTION
**Pull request by diana-macrometa**
This page adds information about the Validations tab. Reviewers, please check my definitions very carefully!

### CODE REVIEW

File: docs/photoniq/p3/view-p3-validations.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+- **Text Analysis** - An analysis of the text content similarity between the optimized and original page versions, assessing changes in text placement, font, and legibility.
+
+### Highlighted Missing Pixels and Optimized
+
+The original and optimized versions are shown side by side. You can scroll down to see both pages. Any discrepancies, such as missing pixels, are highlighted in bright green.
+
+## Functional Validation
+
+Functional validation measures how the optimized page functions compared to the original page. It views the requests sent by the original, takes a random subset of those, and then checks to see if the optimized page is performing the same requests.
```
Comment (diana-macrometa) [inline docs/photoniq/p3/view-p3-validations.md]: Is "requests" the correct word?

Comment (edgargarciamacrometa) [reply docs/photoniq/p3/view-p3-validations.md]: @diana-macrometa - Apart from doing network request comparison, the functional testing checks other aspects of website functionality, such as if new errors are introduced in the optimized version, also if elements such as links, buttons, navigation are working as expected.

File: docs/photoniq/p3/view-p3-validations.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+- **Structural Similarity Analysis** - A measure that evaluates the similarity in structure between the optimized and original page versions, focusing on changes that affect perceived quality.
+
+- **Template Matching** - Compares specific template patterns found in the original page to those in the optimized version to check for alterations.
+
+- **Text Analysis** - An analysis of the text content similarity between the optimized and original page versions, assessing changes in text placement, font, and legibility.
+
+### Highlighted Missing Pixels and Optimized
+
+The original and optimized versions are shown side by side. You can scroll down to see both pages. Any discrepancies, such as missing pixels, are highlighted in bright green.
```
Comment (dwivedianurag) [inline docs/photoniq/p3/view-p3-validations.md]:
```suggestion
The original and optimized versions are shown side by side. You can scroll down to see both pages. Any discrepancies, such as missing pixels, are highlighted in bright green on the original screenshot
```

**Review by dwivedianurag — APPROVED** [review]
ok

**Pull request merged**